config/sql/config.go

```diff
@@ -24,6 +25,8 @@ func coerceFunc(cType coerceType) qsqlio.CoerceFunc {
 	switch cType {
 	case Int64ToBool:
 		return qsqlio.Int64ToBool
+	case StringToFloat:
```

Comment:
This commit is already present in the earlier PR, right? Please drop it from this PR.